msaccess-db-create-dialog-fix.diff: testing...( patch -l -p0  -d /var/tmp/fst/src/ooo-build/build/ooc680-m6 --dry-run ) < /var/tmp/fst/src/ooo-build/patches/src680/./msaccess-db-create-dialog-fix.diff
patching file officecfg/registry/data/org/openoffice/Office/DataAccess.xcu                                                  Reversed (or previously applied) patch detected!  Assume -R? [n]
Apply anyway? [n]                                                                                                           Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file officecfg/registry/data/org/openoffice/Office/DataAccess.xcu.rej          patching file dbaccess/source/ui/misc/dbumiscres.src
Hunk #1 succeeded at 77 (offset -26 lines).                                                                                 patching file dbaccess/source/ui/misc/dsntypes.cxx
Hunk #1 succeeded at 528 (offset -23 lines).                                                                                Testing patch msaccess-db-create-dialog-fix.diff failed. at /var/tmp/fst/src/ooo-build/patches/apply.pl line 54.
make: *** [stamp/patch.apply] Error 1
diff -Naur ooo-build.orig/patches/src680/msaccess-db-create-dialog-fix.diff ooo-build/patches/src680/msaccess-db-create-dialog-fix.diff
--- ooo-build.orig/patches/src680/msaccess-db-create-dialog-fix.diff	2005-08-19 18:43:02.000000000 +0200
+++ ooo-build/patches/src680/msaccess-db-create-dialog-fix.diff	2006-06-20 21:19:46.000000000 +0200
@@ -1,25 +1,3 @@
---- DataAccess.xcu	2005-08-19 13:18:24.000000000 +0530
-+++ officecfg/registry/data/org/openoffice/Office/DataAccess.xcu	2005-08-19 11:24:38.000000000 +0530
-@@ -785,19 +785,6 @@
-       </node>
-     </node>
-   </node>
--	<node oor:name="UserDefinedDriverSettings">
--		<node oor:name="MDB Driver" oor:op="replace">
--			<prop oor:name="DriverPageDisplayName">
--				<value>MDB-SDBC Driver V 0.1</value>
--			</prop>
--			<prop oor:name="DriverTypeDisplayName">
--				<value>mdb</value>
--			</prop>
--			<prop oor:name="DriverDsnPrefix">
--				<value>sdbc:mdb:</value>
--			</prop>
--		</node>
--	</node>
- </oor:component-data>
- 
- 
 --- dbaccess/source/ui/misc/dbumiscres.src	2005-08-19 13:17:54.000000000 +0530
 +++ dbaccess/source/ui/misc/dbumiscres.src	2005-08-18 11:03:31.000000000 +0530
 @@ -103,7 +103,8 @@
